tests/Agent/IntegrationTests/ContainerApplications/docker-compose.yml

```diff
@@ -76,8 +78,49 @@ services:
             service: smoketestapp
         build:
             dockerfile: SmokeTestApp/Dockerfile.centos
+
+    UbuntuX64Kafka1TestApp:
+        extends:
+            service: smoketestapp
```

Comment:
Instead of extending the smoketestapp, maybe it would be better to extract a common service definition that this service and the smoketestapp service can extend? Taking that approach will probably make it easier to understand which things are unique to each app.

Reply:
Yeah, that would be a good idea.  I think it should be a ticket for later though.

Reply:
If you decide to take this on later can you submit a story for it?